lte/gateway/c/oai/tasks/mme_app/mme_app_context.c

```diff
@@ -2224,7 +2224,6 @@ static bool mme_app_recover_timers_for_ue(
   OAILOG_FUNC_RETURN(LOG_MME_APP, false);
 }
 
-
```

Comment:
might want to remove the no-op oai changes to reduce the number of necessary reviewers :)

Reply:
not committed this oai file this time.